RUM-6885 Add the docs for RumMonitor#addViewLoadingTime API (#25914)
DataDog/documentation: content/en/real_user_monitoring/mobile_and_tv_monitoring/advanced_configuration/android.md

{{% /tab %}}
{{< /tabs >}}

### Notify the SDK that your view finished loading

> We are tracking the time it takes for your view to load. To notify the SDK that your view has finished loading you will need to call the `addViewLoadingTime(override=)` method
> through the `GlobalRumMonitor` instance. This method should be called when your view is fully loaded and displayed to the user:

Reviewer: ```suggestion
iOS RUM tracks the time it takes for your view to load. To notify the SDK that your view has finished loading, call the `addViewLoadingTime(override=)` method
through the `GlobalRumMonitor` instance. Call this method when your view is fully loaded and displayed to the user:
```

Not sure who "we" is here, so made some guesses.

{{< tabs >}}
{{% tab "Kotlin" %}}
   ```kotlin
       @OptIn(ExperimentalRumApi::class)
       fun onViewLoaded() {
            GlobalRumMonitor.get().addViewLoadingTime(override = false)
       }
   ```
{{% /tab %}}
{{% tab "Java" %}}
   ```java
       @OptIn(markerClass = ExperimentalRumApi.class)
       public void onViewLoaded() {
            GlobalRumMonitor.get().addViewLoadingTime(override);
       }
   ```
{{% /tab %}}
{{< /tabs >}}

> Use `override` option to replace previously calculated loading time for current view.

Reviewer: ```suggestion
Use the `override` option to replace the previously calculated loading time for the current view.
```

> 
> Once the view loading time is sent, the loading time is accessible as `@view.loading_time` and will be visible also in the RUM UI.

Reviewer: ```suggestion

After the loading time is sent, it is accessible as `@view.loading_time` and is visible in the RUM UI.
```
